### PR TITLE
filter.d/domino-smtp.conf: filter for IBM Domino SMTP Server TASK to detect failed login attempts

### PR DESCRIPTION
**Pull request by cbrandlehner**
Before submitting your PR, please review the following checklist:

- [x] **CHOOSE CORRECT BRANCH**: if filing a bugfix/enhancement
      against 0.9.x series, choose `master` branch
- [ ] **CONSIDER adding a unit test** if your PR resolves an issue
- [ ] **LIST ISSUES** this PR resolves
- [ ] **MAKE SURE** this PR doesn't break existing tests
- [ ] **KEEP PR small** so it could be easily reviewed.
- [ ] **AVOID** making unnecessary stylistic changes in unrelated code
- [x] **ACCOMPANY** each new `failregex` for filter `X` with sample log lines
      within `fail2ban/tests/files/logs/X` file

### CODE REVIEW

File: config/filter.d/domino-smtp.conf [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+#          (?:::f{4,6}:)?(?P<host>\S+)
+# Values:  TEXT
+#
+# Sample log entries (used different time formats and an extra sample with process info in front of date)
+# 01/23/2009 19:54:51   SMTP Server: Authentication failed for user bd ; connecting host 125.120.107.140
+# [28325:00010-3735542592] 22.06.2014 09:56:12   smtp: brian [74.208.72.28] authentication failure using internet password
+# 08.09.2014 06:14:27   smtp: drweb [81.82.251.231] authentication failure using internet password
+# 08.09.2014 06:14:27   SMTP Server: Authentication failed for user drweb ; connecting host 81.82.251.231
+
+failregex = .* SMTP Server: Authentication failed for user .* connecting host <HOST>
```
Comment (sebres) [inline config/filter.d/domino-smtp.conf]: please fix this bad expression (anchors left/right, don't use greedy catch-all):
```regex
__prefix = (?:\[[^\]]+\])?\s+
failregex = ^%(__prefix)sSMTP Server: Authentication failed for user .*? ; connecting host <HOST>$
```

File: config/filter.d/domino-smtp.conf [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+         .*smtp:.* \[<HOST>\] authentication failure using internet password
```
Comment (sebres) [inline config/filter.d/domino-smtp.conf]:
```regex
         ^%(__prefix)ssmtp:.*? \[<HOST>\] authentication failure using internet password\s*$
```

**Comment by sebres** [comment]
BTW. You don't need to always create a new PR for each changes, just use git: checkout local copy (master branch), commit new changes, push as current branch (master) into your cloned repository - all the changes will be right away visible here

**Comment by cbrandlehner** [comment]
Hi!

Thanks for the tip, will try it. No big experience with develeoping in teams yet. ;-)



-----"Serg G. Brester" <notifications@github.com> schrieb: -----
An: fail2ban/fail2ban <fail2ban@noreply.github.com>
Von: "Serg G. Brester" <notifications@github.com>
Datum: 07.11.2016 12:58
Kopie: cbrandlehner <christian@brandlehner.at>, Author <author@noreply.github.com>
Betreff: Re: [fail2ban/fail2ban] Master (#1600)

BTW. You don't need to always create a new PR for each changes, just use git: checkout local copy (master branch), commit new changes, push as current branch (master) into your cloned repository - all the changes will be right away visible here
  &#8212;

**Pull request closed without merging**